release-blocking-jobs.md

```diff
@@ -87,9 +87,31 @@ manual, human interpretation, we choose to move them to a separate dashboard.
 
 CI Signal Team members will regularly check the Informing Dashboard for new
 failures.  Unexplained failures that are not flakes should be considered reason
-to block the release until the failures are investigated and evaluated.  Unlike
-Blocking, however, jobs in Informing do not need to be green before authorizing
-a release, they can be failing for known and "acceptable" reasons.
+to block the release until the failures are investigated and evaluated.
+Unlike Blocking, however, jobs in Informing do not need to be green before
+authorizing a release, they can be failing for known and "acceptable" reasons.
+
+The criteria for deciding whether a failure in informing is to be ignored can be
+based on the following considerations:
+
+* Failure is due to a malfunction of the components that run a job and said
+  components are not related to Kubernetes
+  * For example, if a job is marked as failing because a test cluster could not be setup
+    but there are similar jobs that run the same tests, then the failing job
+    can be ignored.
+  * If a job stage stage, that is not related to the test results, fail
+    (e.g., couldn't upload some logs to a remote source) then the job can be
+    ignored.
+  * If a test-infra-related component fails and no there are no other factors
+    that could be responsible for the failure then the failure can be ignored.
+* Failure well understood and is caused by the configuration of a test and said
+  configuration is not related to an official SLO
+  * For exaple, if a test polls for metrics more fequently than these are
+    updated and fails because of it, then the failure may be ignored.
+* Job is flaking too frequently and is marked as failing
+  * This should only be done when different tests are flaking. If the same set
```

Comment:
If the same set of tests are flaking... ? Is the sentence not finished? Do I understand it correctly that the criteria is that if the same test is flaking as much as it marks the whole job as failing, then it blocks the release?

Reply:
:sweat_smile: sorry about this one. I rewrite it but yes to your last question.
If there are multiple unrelated tests that flake a few times and thus caused the job to be marked as failing then we may ignore, otherwise we block.